posts/2020-03-10-calculating-pi-concurrently.adoc

```diff
@@ -900,13 +921,19 @@ location like shown in the final version of the stream processing:
 [source,scala]
 ----
   val piF: Future[BigDecimal] = indexes
-    .groupBy(Settings.parallelism, _ % Settings.parallelism)
-    .map(piBBPdeaTermI).async
-    .fold(BigDecimal(0)){case (acc, term) => acc + term}
+    .groupBy(maxSubstreams = Settings.parallelism, genKey)
+    .map(index => piBBPdeaTermI(index)).async
+    .via(calculateSum)
     .mergeSubstreams
-    .runWith(sumOfTerms)
+    .via(calculateSum)
+    .runWith(Sink.head)
 ----
 
+If we compare the different implementations of the calculation, we see that the
+`Future` based ones (the _Chunk_ based one, with a sufficiently high number of
+chunks, and the round-robin one) and the Akka Streams based ones, we can 
+conclude that their performance is on par.
```

Comment:
Will you make another nice graph? 😄 

Also wondering if the streams ones exhibit the same performance characteristic as the round-robin ones, and if there's a noticeable difference between the Akka Streams implementations.

Reply:
> Will you make another nice graph? 😄

I've decided not to do that as, ideally, I'd have to run a jmh benchmark that give average calculation times + variances.

This will take too much time.